package.json

```diff
@@ -4,14 +4,17 @@
     "ci": "prettier . -c && eslint . --ext .ts,.tsx",
     "prettier": "prettier . -w",
     "lint": "eslint .  --ext .ts,.tsx --fix",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "generate:client:js": "openapi-generator-cli generate --global-property apis,models,supportingFiles,modelDocs=false -i schemas/openapi.yml -g typescript-fetch -c packages/client/js/openapitools.json -o packages/client/js --additional-properties=generateSourceCodeOnly=true",
+    "generate:client:python": "openapi-generator-cli generate --global-property apis,models,supportingFiles,modelDocs=false -i schemas/openapi.yml -g python-pydantic-v1 -c packages/client/python/openapitools.json -o packages/client/python --library asyncio --additional-properties=generateSourceCodeOnly=true,packageName=agent_protocol_client"
```

Comment:
Running both this and the existing python client generate script (via [make](https://github.com/AI-Engineer-Foundation/agent-protocol/blob/2b986e06dd2e616ea28116f190c9f067c03cc82d/packages/client/python/Makefile)) modifies the python client.
At a glance, it appears to just be styling changes, but I'm not certain.